packages/mobile/src/fiatExchanges/ProviderOptionsScreen.tsx

```diff
@@ -127,42 +128,58 @@ function ProviderOptionsScreen({ route, navigation }: Props) {
     SIMPLEX_RESTRICTED,
     RAMP_RESTRICTED,
     TRANSAK_RESTRICTED,
+    XANPOOL_RESTRICTED,
   } = getProviderAvailability(userLocation)
 
+  const xanpool = {
+    id: CicoProviderNames.Xanpool,
+    paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
+    restricted: XANPOOL_RESTRICTED,
+    onSelected: () =>
+      navigate(Screens.XanpoolScreen, {
+        localAmount: route.params.amount,
+        currencyCode: localCurrency,
+        currencyToBuy: selectedCurrency,
+      }),
+  }
+
+  const moonpay = {
+    id: CicoProviderNames.Moonpay,
+    paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
+    restricted: MOONPAY_RESTRICTED,
+    onSelected: () =>
+      openMoonpay(route.params.amount, localCurrency || FALLBACK_CURRENCY, selectedCurrency),
+  }
+
+  const simplex = {
+    id: CicoProviderNames.Simplex,
+    paymentMethods: [PaymentMethod.Card],
+    restricted: SIMPLEX_RESTRICTED,
+    onSelected: () => openSimplex(account),
+  }
+
+  const ramp = {
+    id: CicoProviderNames.Ramp,
+    paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
+    restricted: RAMP_RESTRICTED,
+    onSelected: () =>
+      openRamp(route.params.amount, localCurrency || FALLBACK_CURRENCY, selectedCurrency),
+  }
+
+  const transak = {
+    id: CicoProviderNames.Transak,
+    paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
+    restricted: TRANSAK_RESTRICTED,
+    onSelected: () =>
+      openTransak(route.params.amount, localCurrency || FALLBACK_CURRENCY, selectedCurrency),
+  }
+
   const providers: {
     cashOut: CicoProvider[]
     cashIn: CicoProvider[]
   } = {
-    cashOut: [],
-    cashIn: [
-      {
-        id: CicoProviderNames.Moonpay,
-        paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
-        restricted: MOONPAY_RESTRICTED,
-        onSelected: () =>
-          openMoonpay(route.params.amount, localCurrency || FALLBACK_CURRENCY, selectedCurrency),
-      },
-      {
-        id: CicoProviderNames.Simplex,
-        paymentMethods: [PaymentMethod.Card],
-        restricted: SIMPLEX_RESTRICTED,
-        onSelected: () => openSimplex(account),
-      },
-      {
-        id: CicoProviderNames.Ramp,
-        paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
-        restricted: RAMP_RESTRICTED,
-        onSelected: () =>
-          openRamp(route.params.amount, localCurrency || FALLBACK_CURRENCY, selectedCurrency),
-      },
-      {
-        id: CicoProviderNames.Transak,
-        paymentMethods: [PaymentMethod.Card, PaymentMethod.Bank],
-        restricted: TRANSAK_RESTRICTED,
-        onSelected: () =>
-          openTransak(route.params.amount, localCurrency || FALLBACK_CURRENCY, selectedCurrency),
-      },
-    ].sort(sortProviders),
+    cashOut: [xanpool],
```

Comment:
Even if it's a single item I would do the `.sort(sortProviders)` thing here as well to future-proof it.